napari/utils/colormaps/colormap_utils.py

```diff
@@ -299,6 +350,7 @@ def label_colormap(num_colors=256, seed=0.5):
     colors[0, :] = 0  # ensure alpha is 0 for label 0
     return Colormap(
         name='label_colormap',
+        display_name='label_colormap',  # TODO: needs translation?
```

Comment:
Should this be `label colormap` and also have a translation @napari/core-devs ?

Reply:
Yes we should, maybe @jni has a good idea for a name?

Reply:
Not really! LOL "random colors"? "low discrepancy colors"? "random L*a*b* colors"/"random CIELAB colors"?

Reply:
How did you prevent label colormap from showing in the list of selectable colormaps for a image layer?